website1/src/scss/_components.scss

```diff
@@ -45,17 +91,88 @@
     width: 190px;
     background-color: $yellow;
   }
-}
 
-.title-description {
-  max-width: 626px;
-  width: 100%;
-  line-height: 159%;
-  margin-top: 28px;
+  &-description {
+    max-width: 626px;
```

Comment:
It's the universal component, are you sure should be limited by width?
I think if this component describes behavior as `text`, it should contain only styles for text.

Reply:
Yes, it is by design.